src/screens/AnswerScreen/AnswerScreen.js

```diff
@@ -0,0 +1,293 @@
+import React from 'react';
+import {View, Text, SafeAreaView, Image, TouchableOpacity, TextInput, FlatList, Alert, Linking, ColorPropType} from 'react-native';
+import { firebase } from '../../config/config';
+import Icon from 'react-native-vector-icons/Ionicons';
+import { LOGO } from '../../config/styles.js';
+import styles from './styles';
+import moment from 'moment';
+import ImagePicker from 'react-native-image-crop-picker';
+import * as Progress from 'react-native-progress';
+
+class AnswerScreen extends React.Component {
+    userUid = firebase.auth().currentUser.uid
+    state = {answerQuery : '', answerDetail: [], questionName: '', questionAskId : '', likeButton: false, queryImage: null, questionId: '', transferred: 0, setImage: null, setDisplayImage: null, answerImage: null, answerPostNumber: null};
```

Comment:
Normally the maximum characters that would be in a sing line of code is 120 so that is easy to review and no need to scroll horizontally.